### PR TITLE
feat: (IAC-897) Annotate NGINX to indicate the desired subnets to provision LBs internal IP addrs from

### PR DESCRIPTION
**Pull request by dhoucgitter**
### Changes

* Add documentation for the added AWS specific configuration variable V4_CFG_AWS_LB_SUBNETS
* The V4_CFG_AWS_LB_SUBNETS values specifies the AWS subnets and by association the AWS availability zones to deploy the load balancing service to. Specifying a subnet value or values for this variable supercedes the Subnet Discovery method decribed in [AWS docs](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html) and eliminates the need to tag subnets as described in scenario 2 of this [table](https://github.com/sassoftware/viya4-iac-aws/blob/main/docs/user/BYOnetwork.md#supported-scenarios-and-requirements-for-using-existing-network-resources) 

### Tests

| Scenario | Provider | method | kubernetes_version | cluster_api_mode | V4_CFG_AWS_LB_SUBNETS  | notes |
|----------|----------|--------|--------------------|--|-------|-|
| 1 | AWS | Ansible | 1.27 | public | unset | AWS NLB AZs correspond to public tagged subnets via Subnet Discovery
| 2 | AWS | Ansible | 1.27 | public | "subnet-034f8eaf7b0c8a744" | AWS NLB shows AZ was set to us-west-1a and it's private IP address is assigned from CIDR 192.168.130.32/28 which is the CIDR for subnet-033f8eaf7b0c8a744 
| 3 | AWS | Ansible | 1.27 | public | "subnet-033f8eaf7b0c8a744, subnet-0c20ecd67feefbee7" | AWS NLB shows AZs set to us-west-1a and us-west-1c and it's private IP address is assigned from CIDR 192.168.130.32/28 and CIDR 192.168.130.48/28 which are the CIDRs for subnet-033f8eaf7b0c8a744 and subnet-0c20ecd67feefbee7 respectively 
| 4 | AWS | Ansible | 1.27 | private, Environment setup using gitlab aws byo private-cluster.tfvars | unset | AWS NLB chooses AZs us-west-1c and us-west-1a from the public and private subnets. This is existing behavior and is not influenced by the new config variable. 
| 5 | AWS | Ansible | 1.27 | private, Environment setup using gitlab aws byo private-cluster.tfvars | subnet-04bbcb82452df7259 | AWS NLB chooses AZ us-west-1a from the specified subnet-04bbcb82452df7259.
| 6 | AWS | Ansible | 1.27 | private, Environment setup using gitlab aws byo private-cluster.tfvars | subnet-0efe7b1a630e947f3, subnet-013a0823b2666d44d | AWS NLB chooses AZs us-west-1c and us-west-1a and it's private IP address is assigned from CIDR 10.10.128.128/25 and 10.10.128.0/25 which are the CIDRs for subnet-0efe7b1a630e947f3 and subnet-013a0823b2666d44d respectively

### CODE REVIEW

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -155,6 +156,13 @@ When V4_CFG_MANAGE_STORAGE is set to `true`, the `sas` and `pg-storage` storage
...
 | V4_CFG_INGRESS_MODE | Whether to create a public or private Loadbalancer endpoint | string | "public" | false | Possible values: "public", "private". Setting this option to "private" adds options to the ingress controller that create a LoadBalancer with private IP address(es) only. | baseline |
 
+## Load Balancer
+
+| Name | Description | Type | Default | Required | Notes | Tasks |
+| :--- | ---: | ---: | ---: | ---: | ---: | ---: |
+| V4_CFG_AWS_LB_SUBNETS | The AWS subnets and by association the AWS availability zones to deploy the load balancing service to. Specifying a subnet value or values for this variable takes precedence over the Subnet Discovery method described in [AWS docs](https://docs.aws.amazon.com/eks/latest/userguide/network-load-balancing.html) and eliminates the need to tag AWS subnets as documented in scenario 2 of this [table](https://github.com/sassoftware/viya4-iac-aws/blob/main/docs/user/BYOnetwork.md#supported-scenarios-and-requirements-for-using-existing-network-resources) | string | | false | The value is either a comma separated list of subnet IDs, or a comma separated list of subnet names. | baseline |
+
+
```
Comment (thpang) [inline docs/CONFIG-VARS.md:165]: Is this not a BYO networking type item? Seems like it belongs in another location.

Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:165]: It is most likely a BYO networking concern although it is not limited to just some BYO networking scenario. Can also be used with BYON=0 which equates to not bringing your own network. I put it under its own heading as I didn't think it belonged in any existing section, I will see if I can find an alternative to suggest.

Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:165]: @thpang, we don't actually have any existing BYO doc in the DAC repo (the link I have in that section points back to the AWS repo's BYOnetwork.md doc, so it's all back in each IAC project currently.

Comment (dhoucgitter) [reply docs/CONFIG-VARS.md:165]: I am going to add to the description that this variable can be used with any BYO networking scenario 0-3.

**Review by riragh — APPROVED** [review]

**Review by jarpat — APPROVED** [review]

**Pull request merged**